frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx

```diff
@@ -147,12 +152,15 @@ function EmbedFrame({
         {hasHeader && (
           <Header className="EmbedFrame-header">
             {finalName && (
-              <TitleAndDescription
-                title={finalName}
-                description={description}
-                className="my2"
-              />
+              <TitleAndDescriptionContainer>
+                <TitleAndDescription
+                  title={finalName}
+                  description={description}
+                  className="my2"
+                />
+              </TitleAndDescriptionContainer>
             )}
+            {isValidElement(header) ? header : null}
```

Comment:
not sure if this is a bit overkill. Since we're controlling what's being passed `StyledDashboardTabs` in this case. Would defaulting the `header` or (`dashboardTabs` as per Jesse's suggestion) to `null` and render simply render this better?

e.g.

```suggestion
            {dashboardTabs}
```

And in the component props
```ts
...
children,
dashboardTabs = null,
name,
...
```